src/Tribe/RSVP.php

```diff
@@ -1370,6 +1370,12 @@ public function send_tickets_email( $order_id, $event_id = null ) {
 		if ( $sent ) {
 			foreach ( $all_attendees as $attendee ) {
 				$this->update_ticket_sent_counter( $attendee['qr_ticket_id'], $this->attendee_ticket_sent );
+
+				$this->update_attendee_activity_log( $attendee['attendee_id'], [
+					'type'  => 'email',
+					'name'  => $attendee['holder_name'],
+					'email' => $attendee['holder_email'],
+				] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 85aa8284204f8d42e49bdc3aab3b9103</sub>